pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -109,6 +110,13 @@ type Patroni struct {
 	Slots                map[string]map[string]string `json:"slots"`
 }
 
+//StandbyCluster
+type StandbyDescription struct {
+	ClusterName string `json:"cluster,omitempty"`
+	UID         string `json:"uid,omitempty"`
```

Comment:
is it the UID of the source cluster from where the standby cluster is streaming ?

Reply:
yes